static/model_config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "in": {
```

Comment:
There seems to be much duplication between in and out - I still don't fully understand the purpose of all fields. I think .json allows comments.

Reply:
https://www.quora.com/How-do-I-write-comments-inside-a-JSON-document

Totally agree; I wanted to flesh out object reuse and multi-model support before I paired it down a bit. I've added a TODO to model_config.py to dedupe In and Out model configs.

Reply:
I've pared this down to just be for fetching; it's not longer involved in classification.